web/manifest/handler.go

```diff
@@ -0,0 +1,71 @@
+package manifest
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+type manifest struct {
+	ShortName       string `json:"short_name"`
+	Name            string `json:"name"`
+	Description     string `json:"description"`
+	StartURL        string `json:"start_url"`
+	Orientation     string `json:"orientation"`
+	BackgroundColor string `json:"background_color"`
+	ThemeColor      string `json:"theme_color"`
+	Display         string `json:"display"`
+	Icons           []icon `json:"icons"`
+}
+
+type icon struct {
+	Src   string `json:"src"`
+	Type  string `json:"type"`
+	Sizes string `json:"sizes"`
+}
+
+// Handler provides a way to serve Web App Manifests.
+// A web app manifest is a simple JSON file that controls how our app appears to
+// the user in areas where they would expect to see apps (for example, a mobile device's
+// home screen), direct what the user can launch, and define its appearance at launch.
+//
+// For further details see:
+// * https://www.w3.org/TR/appmanifest/
+// * https://developer.mozilla.org/en-US/docs/Web/Manifest
+// * https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
+type Handler interface {
+	ServeHTTP(w http.ResponseWriter, r *http.Request)
+}
+
+type handler struct {
+	m manifest
+}
+
+// NewHandler creates a new handler that is able to serve web app manifest.
+func NewHandler() Handler {
+	return &handler{
+		m: manifest{
+			ShortName:       "Concourse",
```

Comment:
All this stuff is hard-coded which is hopefully o.k. for now.

The ability to change certain parameters may be added in the future, though-- that's why I decided to go with a "possibly dynamic approach" of creating the web app manifest instead of dropping a static JSON file into the "/public/" folder.